app/lib/common/providers/common_providers.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:acter/common/models/profile_data.dart';
```

Comment:
isn't this filename a bit too much of a catch-all? like account, network state and loading, chat and profile data are clearly not the same category of things ...

Reply:
Yes, I agree with not belonging to same categories but I named it as in the specifics of containing the providers which are global in application layout and is intended to be used in more than one features.